web/modules/custom/hoeringsportal_citizen_proposal/hoeringsportal_citizen_proposal.module

```diff
@@ -61,3 +61,10 @@ function hoeringsportal_citizen_proposal_page_attachments(array &$page) {
 function hoeringsportal_citizen_proposal_node_presave(EntityInterface $entity) {
   Drupal::service(Helper::class)->nodeEntityPresave($entity);
 }
+
+/**
+ * Implements hook_cron().
+ */
+function hoeringsportal_citizen_proposal_cron() {
+  Drupal::service(Helper::class)->cron();
+}
```

Comment:
For consistency with similar tasks in this project, e.g. https://github.com/itk-dev/hoeringsportal/blob/develop/web/modules/custom/hoeringsportal_public_meeting/README.md, we should do this outside the Drupal cron schedule.

Reply:
Removed and changed Readme.md